transform/transformers/cord/ecommerce_transformer.py

```diff
@@ -369,3 +367,229 @@ def create_zip(self, img_seq=None):
         response_json_name = CORDFormatter.response_json_name(self.ids.survey_id, self.ids.seq_nr)
         self.image_transformer.zip.append(os.path.join(SDX_RESPONSE_JSON_PATH, response_json_name), json.dumps(self.response))
         bound_logger.info("Sucessfully added json response to zip")
+
+
+class Ecommerce2019Transformer(EcommerceTransformer):
```

Comment:
A little concerned that we might do something stupid like modify or "clean up" `EcommerceTransformer` in the distant future but seems like the most sensible way to do this now (and at least it should fail fast if we do)